demo/tests/test_end_point.py

```diff
@@ -328,6 +328,62 @@ class TestForm(FormidableForm):
         with django_perf_rec.record(path='perfs/'):
             serializer.data
 
+    def test_presets(self):
+
+        form = Formidable.objects.create(label='my_test_form')
+
+        preset1 = form.presets.create(slug='foo', message='message1')
```

Comment:
Actually, I expect to have a presets which actually working in order to avoid future explosion when we will add check on existing Presets (Hell yeah, I'm confident !)

Please create a comparison Presets with three arguments, 

The field argument, with its little name "left" , the operator argument with its little name, "operator" and the last one but not least the value you want to compare.